src/which.js

```diff
@@ -2,6 +2,9 @@ var common = require('./common');
 var fs = require('fs');
 var path = require('path');
 
+// XP's system default value for PATHEXT system variable
```

Comment:
^ Good point. But yes, I also vote we keep it for no other reason than extra safety measure.